### ISSUE TITLE
[ admin ] Finish v0.7.0 docs

### ISSUE THREAD

**Issue by CodingCellist** [idris-lang/Idris2#3173]
The only item remaining on the checklist for releases is:

- [ ] Make libdocs and upload to idris-lang.org

I can make the docs without any issues. However, there was some confusion about which Idris2 website is the "correct" one; whether there's an administrative change going on between some of them; who needs to switch what; etc. So I'm not too sure where to submit the new docs...

I'm opening this issue so that we don't forget  : )

**Comment by andrevidela** [comment]
I just realised this got closed automatically with the checklist being cleared. what's the status on the tarball @CodingCellist ?

**Comment by CodingCellist** [comment]
> I just realised this got closed automatically with the checklist being cleared. what's the status on the tarball [@CodingCellist](https://github.com/CodingCellist) ?

Status is that it is on both under this repo's "Releases" section, as well as available to download from the website. Now, having just had the unfortunate thought and against better judgement decided to confirm it, here's a minor headache: the two tarballs have different sha256 hashes, despite allegedly being the exact same source code... :facepalm:

Which I guess just brings us back to a point in the original issue text:

> it might be better to ask them to move to the GH sources so that there isn't duplicated work?

Can I remember which package managers use the website as a source of truth? Nope, that was back in 2023! Although by now `pack` has won anyhow, and I _believe_ it uses GH to bootstrap (not even the releases iirc, it just checks out the repo).

tl;dr: status is the tarballs are up, but we might want to point the website at the GH ones, just to keep everything nicely in sync? The main counter-argument I can think of, is that the website allows us to easily archive an independent version (both of the site and the releases), but I don't know how much of a concern that is

**Comment by andrevidela** [comment]
pointing to the GH one might be the move. Less infrastructure for us, and always in sync, despite our best efforts to mess it up we won't be able to

**Comment by jfdm** [comment]
If it helps, I would also vote for using the GitHub release pages as the primary archive for now. 

In fact, when maintaining the labs for teaching, I am periodically asked for which Idris2 to install. I point our IT support towards the GitHub releases page.

It would also be good to maintain the separate archive on the idris website to have a separate source of truth. That being said, if the website is not on Github infrastructure, then it may not be that good for redundancy.
